.travis.yml

```diff
@@ -14,7 +14,7 @@ cache: apt
 before_install:
   - sudo add-apt-repository -y "deb http://archive.ubuntu.com/ubuntu/ trusty main universe"
   - sudo apt-get update -qq
-  - sudo apt-get install automake autoconf intltool xsltproc libglib2.0-dev libpolkit-gobject-1-dev libpolkit-agent-1-dev phantomjs gtk-doc-tools libjson-glib-dev libnm-glib-dev libpam0g-dev libssh-dev libsystemd-daemon-dev libsystemd-journal-dev libjson-perl libjavascript-minifier-xs-perl pkg-config glib-networking valgrind libkeyutils-dev xmlto xsltproc libpcp3-dev libpcp-pmda3-dev libpcp-import1-dev pcp liblvm2-dev libgudev-1.0-dev libgirepository1.0-dev gdb
+  - sudo apt-get install automake autoconf intltool xsltproc libglib2.0-dev libpolkit-gobject-1-dev libpolkit-agent-1-dev gtk-doc-tools libjson-glib-dev libnm-glib-dev libpam0g-dev libssh-dev libsystemd-daemon-dev libsystemd-journal-dev libjson-perl libjavascript-minifier-xs-perl pkg-config glib-networking valgrind libkeyutils-dev xmlto xsltproc libpcp3-dev libpcp-pmda3-dev libpcp-import1-dev pcp liblvm2-dev libgudev-1.0-dev libgirepository1.0-dev gdb
```

Comment:
If we're touching this anyway, can we do the same we did to the other scripts and make it one dependency per line (ideally sorted)?
This will vastly improve readability of changes in the future.